lib/node_modules/@stdlib/repl/lib/complete_require.js

```diff
@@ -157,6 +164,87 @@ function complete( out, path, paths, exts ) {
 			}
 		}
 	}
+	out.sort();
+	if ( !isFuzzy ) {
+		return filter;
+	}
+	// Start searching for fuzzy completions...
```

Comment:
This isn't as potentially perf prohibitive as `fs` fuzzy auto-completion, but still carries some perf risk.